bluetooth/src/commonMain/kotlin/device/DeviceConnectionManager.kt

```diff
@@ -63,6 +69,8 @@ abstract class BaseDeviceConnectionManager(
         }
     }
 
+    fun handleNewMtu(mtu: Int) = _mtu.set(mtu)
```

Comment:
Should it check if the `mtu` int is a valid value?

Reply:
I believe in the end system does this check